examples/7-Extra-Simulation/Multiple-protocols.ipynb

```diff
@@ -0,0 +1,1589 @@
IBg4ceDqnijkyMzOBXf4VVZ9Yu4Py/GsYV6JU48LfK0o4cKIEL81aX7MVg8T86eCw6WcliWQ3d4XLi/xiJ/KKXcgvdmLWrzsqP+gMh+5Ngqi7WAWt1OMA47B3Wgn3VjmrgMR/bBG28L/nnntw7bXXYsCAARAE/wCCQYMGYdCgQdixYwe2bNmCvLw8AECDBg3QpUsX5OSQWD195Kj/c2s247neXaNsS/2hgtVMBO94fjmemLbqtOtJiLPi7QcvrLwgUSOczsvO4xVxoqACx/PK8e7cLTVsWc1BL3OCIATGIM8XxQAwMHCM/3kPppzlTycm8R87hC38Fy5ciEWLFqFBgwYYMWIERo4cidatW6v7c3JySOTXMC3dR3DQ3gzKj9XNaEXfGsGXPRXvDV0sGJxz7D1ajJc+rV5UnzHg48foAVvbfDJxsG52H8A/24/ZC49zjpJyD47lleF4frm6CmZtUdlLmNJQCIIIB7PJV5r3uAvIK9dtI/EfG4Qt/Fu2bImDBw8iNzcXH330ET766CN0794d1157LS699FLEx8dXXglRJe7uNwhPrFXEhvzDpZSfmiNNrJryP5FfjserEdmnB2ndJ7BBEAlsVgEfPDIw4uchCIKojA22wwCMa9mECoYQ9YMqzeO/du1afPvtt/j1119RUVEhV8AYEhIScNlll+Gaa67BOeecEylbYwptFO+JtTvhH+TL1X+pAVA9/rV2M5yIQ7/dS3BhyfKQ03JKEscdkxZVqX56YNbtaPRdbyyGJ0I21eZ3X5d9XF8gH9cO5OfI49cRHDkLFoA1saC8YhPWFF4T9Bh6l1WNM2Ue/2ot4FVWVoaffvoJc+bMwYYNG+SKfF1Jbdq0wahRo3DVVVchI6PmV0aNFbQ3zvsrFmlSfhQ4SPxXD0X4d9m1GsNLfzcV/l5Rwp2vLw6rvowUB964p1/NGnmGcya8yKsb5R99SQ4GdW9Ww9ZUnTPBx2c65OPagfwcebTCv5dnA/63tCGeLd2Is178J+77z1KUO4P3gFMDIDzqtfDXsm/fPnzzzTeYN2+eOqMPYwwWiwWDBw/GyJEjMWDAgKALfBHmBN44X/w6F5szOkLJ95eh6H91UIR/h+2rcU3ibt2qvRLnuOO18CL89DAMDr3IIw/5OPKQj2sH8nPk0WcOSCjIn4mXjl6Lp1t8jneGvgGg8mAIvfNCEzPCX0GSJCxduhTffPMNFi9eDK/Xq4r9hg0bYsSIEbjmmmvQsmXLmjhdvSfYjSPn/GvFP6BtADx4VjIaNWpaCxaeuSjCv+vulbjxxlvU7Xe8thBSJb8GevCFB73IIw/5OPKQj2sH8nPkeWrNNkhMGdbJUVT8IZqfGIARC79Bs/NtSL79QwCoNPoP0HswGDEn/LXk5+dj3rx5+Pbbb7Frl7wgDWMMjDFs3bq1pk9XLwl14/jFP2Bc4MvfCBjlPYke5/ePkIVnLs+v3QIXHOi5bylGXnd7WGk99KCrGvQijzzk48hDPq4dyM+Rp8JVgRf+OgQl3af3999hXlIvvNE4XRf1Vwg3FZLejX5iWvgrSJKEd999F++++y4kSQJjDNu2bYvU6eoV4dw4oRsAAHTr83E83LoBGmRm1pCFZy6q8N+9EgMvvx6PvLsyaFl6qFUPepFHHvJx5CEf1w7k58hjtQqY8Md2KMIf4Cgu+hDtiy/AdXtceLH7WoP4B8JvAPxjaHsM7dmiRm0+0zhThP9prdwbjAMHDuDbb7/F3LlzdSv5Ohw0D31NouT1v7F6NfKFdM0es8YAw7/35QP78n2fJLzUq0Ot2FlXYVwKKvpJ8BMEQRD1CyUYBjQqqwAARhdJREFUKGuDlNRx2LTuCNKzduP+L3NxLx7B5KaXwd7J//5T3oWVNQBm/74Ls3/fpX7++LFBNLazjlJjEf+Kigr89NNP+Pbbb9WZfpSqO3bsiFGjRmH48OFISUmpidPVe6rbYpRnAGqKQNFvjr5H4CZrCTp371nlc55pKBH/HruW4aeDrQz7SfSfPhTBizzk48hDPq4dyM+Rx2oV8MtnX2Fh27OhmyBE4ji+6AjeaJyOkyumYOoNmXh1b27Iaa6rMyPaB48MgM1qqZbtZwpnSsT/tIX/unXrdHP7K9WlpKTgiiuuwKhRo9CpU6fTOUVMUlM3jj4dSCGchgAQJzlxwck9GDx8xGnbUZfwC/+l+Omgf/VpEvw1B73IIw/5OPKQj2sH8nPkUXw87qd1CJwdUJI8OLnoRJXEv8LpLH740rg+yEyNh80anmCt69Rr4X/y5El89913mDNnDg4ePAhAju4zxtC7d2+MGjUKl1xyCex2e1WrJnxE6sapWkNAQX+LCBDRY+8apCeloVuffmfcuAE1x3/7YvxwpC0AEv01Db3IIw/5OPKQj2sH8nPkUXw87aWXsLb7JTCKf46Ti45gaNZe9Fi+XBX/AMJqAAA1uwL6/SO7IjXJgbQkO5IT7GdE46DeCX+Px4MFCxZgzpw5WLFiBSRJUqP7WVlZGDFiBEaOHIkWLWJ7cEdNUVs3jnlDAEG2mWF++yRIZWiftweCxwOLxwsminBYrYiPS0ajps2Q1aJV1BoMasR/52L8dKgtif4IQC/yyEM+jjzk49qB/Bx5tD6e/tH72NZ5AIzrAnEcX3AE/+j8F3rkDtBF/xXCbQQANdsQCIer+7eG3WqBzSrAbhVgswkQGJP/E5R/AcE3y6TAAEGQ/+acQ+JyEJtzgMP3r2a7LzPKt833n8QhSfIaQIwBF3RvARvj9UP4n3feeSgqKgIgX7TVasXgwYMxatQo9O/fnwZx1DDRfAAGbwwgxPaqErHJpMLAv4jJ8QVHSPhHAHqRRx7yceQhH9cO5OfIE+jjHatexEzLtTBbFFSQvDi66DgGZu3G5bwXTq2Ygqk3NkSqV8TjB/LVOpNu/QDMVrVJW2q7MRAtaltXRET4d+ggzwDTrl07jBo1CldeeSUyMjKqbyURkrr4AAzdINBypjQCOV7ulR1
```

Comment:
this is called duration, but is used as stim_end. Could you change the name of this variable please?

Reply:
It is not pointing to the correct line of code so i am not sure where it is

Reply:
Ok. I was afraid of confusion with stimulus duration  (which can be different from total simulation duration that we have here), but since this is EPSP and we don't have any step stimulus, keeping duration is ok.